File-Formats/DocIO/Working-with-Charts.md

```diff
@@ -759,6 +759,82 @@ document.Close()
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Applying-3D-formats).
 
+## Removing Chart Title
+
+The following code example illustrates how to remove the chart title from the document.
```

Comment:
The following code example illustrates how to remove the chart title from the Word document.

Reply:
Addressed the feedback